src/resources/parser/texture/img.js

```diff
@@ -18,7 +18,9 @@ class ImgParser {
         // by default don't try cross-origin, because some browsers send different cookies (e.g. safari) if this is set.
         this.crossOrigin = registry.prefix ? 'anonymous' : null;
         this.maxRetries = 0;
-        // disable ImageBitmap
+        // As of today (9 Jul 2021) ImageBitmap only works on Chrome:
```

Comment:
please create an issue for this in the engine repo (if not already) so that we can reevaluate at a later state.

Reply:
done 👍🏻

https://github.com/playcanvas/engine/issues/3336